UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Common/Logging/EventIds.cs

```diff
@@ -655,10 +655,15 @@ public enum EventIds
         /// </summary>
         ProductsFetchedFromESS = 809129,
 
+        /// <summary>
+        /// 809130 - Empty batch response from ESS.
+        /// </summary>
+        EmptyBatchResponse = 809130,
+
         /// <summary>
         /// 809130 - Macro is invalid or unavailable.
```

Comment:
Events sequence number need to be corrected
```suggestion
        /// 809131 - Macro is invalid or unavailable.
```

Reply:
Resolved.